spec/models/appointment_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe Appointment, type: :model do
+  before(:each) do
+    @user = User.create(name: 'John Doe')
```

Comment:
Great effort testing the appointment table validations, however, I think it would have been better if you had created the appointment before each appointment test block , I will follow up down bellow on this.